images/Makefile

```diff
@@ -16,7 +16,7 @@ include image.mk
 
 PLATFORMS ?= linux_amd64 linux_arm64
 
-CEPH_BASE_IMAGE ?= ceph/daemon-base:v3.0.5-stable-3.0-luminous-centos-7-$(PLATFORM_ARCH)
```

Comment:
why was this pulling amd64? `PLATFORM_ARCH` would have been set to `arm64` when cross building right?

Reply:
isn't the Makefile evaluated on the build host before the cross build is started? So the platform arch would be set to the build machine's arch. Or maybe i'm misunderstanding the cross build.

Reply:
We explicitly pass `PLATFORM` here https://github.com/rook/rook/blob/master/images/Makefile#L33 when cross building. I would expect this to be set correctly. I wonder if there is something else going on here. Does your fix actually fix the issue?

Reply:
i opened the PR optimistically and was just verifying locally. Looks like it isn't working, back to the drawing board. Note that we need `PLATFORM_ARCH` set in [image.mk](https://github.com/rook/rook/blob/master/images/image.mk#L34).

Reply:
@bassam if I add a check in the `Makefile` for `$(PLATFORM)`, it is always `linux_amd64`. I am not seeing yet why `linux_arm64` is not set even though I see the arm64 `go` and `docker` builds triggered